### ISSUE TITLE
Alignment seems off

### ISSUE THREAD

**Issue by vmax77** [Barteling/STweep#65]
**Describe the bug**
The declaration alignments do not seem to respect the setting.

**Expected behavior**
Preformatting: 
![image](https://user-images.githubusercontent.com/4384278/182169613-c104cf54-3fbe-4aeb-8b8d-69b044b02613.png)

Post Formatting 
![image](https://user-images.githubusercontent.com/4384278/182169708-acdfb0ad-884e-4473-a7ab-5398961d283a.png)

I would have expected the declarations to be aligned at one column and the assignments to be aligned further down. But it rather did the opposite. 

**STweep version info**
STweep for TwinCAT 4.0.4

**IDE version info**
Visual Studio Professional 2019

**Additional context**
![Screenshot 2022-08-01 162033](https://user-images.githubusercontent.com/4384278/182169145-3c6733bd-6c7b-4f9f-aa27-23cef63f32f1.png)
[VV STweep.zip](https://github.com/STweep/STweep/files/9234644/VV.STweep.zip)

**Comment by Barteling** [comment]
Hmm operation cancelled exception probably means i have hit my time out safety limit.
I'll dig into it.

could you post the code above in text form? saves me a bunch of typing ^^

**Comment by vmax77** [comment]
Sure thing 


```

PROGRAM PRG_TestrigMonitoring
VAR

	fChamberPressure : REAL;

	tonFeedbackResponseTimer : TON;
	tFeedbackResponseTimeout : TIME := T#5S;
	rtrig_FeedbackMismatch	 : R_TRIG;
	arrValveFeedbackMatch	 : ARRAY [1..GVL_CHNL_CONFIG.CONST.NUM_DO_CHANNELS] OF BOOL;
	nDOChnlIdx			 : UINT;
	nFbckCCIdx			 : UINT;
	nFbckDIIdx			 : UINT;
	nValveMismatchCnt		 : INT;
	nValveMismatchIdx		 : INT;

	/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
	// Thresholds
	fChamberPressureThresholdHigh : REAL := 500;
	fChamberPressureThresholdLow	: REAL := 1E-03;
	fTempThreshold			: REAL := -180;
	fPI1_1_Threshold			: REAL;
END_VAR

```

**Comment by vmax77** [comment]
I hit format code on the entire PRG, the alignment problem still exists but the error gets generated while formatting this method


```
(* general parameters 													 *)
stControllerParameter.iMode					:= 1;								(* 1=heating, 2=cooling, 3=heating&cooling *)
stControllerParameter.iReactionOnFailure	:= eCTRL_ReactionOnFailure_StopController;	(* 0=controller off, 1=manual op, 2=yMin, 3=yMax *)

stControllerParameter.bSelCtrlParameterSet 	:= FALSE; (* FALSE = internal set, TRUE = external set *)

stControllerParameter.dwAlarmSupp			:= 16#0000EFFF;	(* alarm suppression *)
stControllerParameter.tCtrlCycleTime		:= T#500MS;		(* controller cycle time *)
stControllerParameter.tTaskCycleTime		:= T#1MS;		(* plc task cycle time *)

(* tuning parameters 													 *)
stControllerParameter.iTuningMode := 1; (* only heating, only cooling, first heating then cooling or vice versa *)

stControllerParameter.tTuneStabilisation := T#20S; (* wait for a stable system *)

stControllerParameter.fEndTunePercentHeating	:= 80.0;	(* switch to closed loop control when X > 0.8*W *)
stControllerParameter.fYTuneHeating			:= 100.0;	(* step change while tuning operation *)
stControllerParameter.fYStableHeating		:= 0.0;	(* tuning operation *)

stControllerParameter.fEndTunePercentCooling	:= 20.0;	(* switch to closed loop control when X < 0.2*W *)
stControllerParameter.fYTuneCooling			:= 0.0;	(* step change while tuning operation *)
stControllerParameter.fYStableCooling		:= 0.0;	(* tuning operation *)
stControllerParameter.fScalingFactor		:= 1.0;	(* Scaling factor for KP heating/cooling *)

(* setpoint parameters 													 *)
stControllerParameter.fWMin	:= 0.0;	(* lower limit *)
stControllerParameter.fWMax	:= 100.0;	(* upper limit *)

(* start up *)
stControllerParameter.bEnableSoftStart	:= FALSE;	(* FALSE = no soft start, TRUE = soft start *)
stControllerParameter.bEnableRamping	:= TRUE;	(* FALSE = no ramping, TRUE = ramping *)

stControllerParameter.fWStartUp		:= 0.0;	(* soft start plateau setpoint *)
stControllerParameter.tStartUp		:= T#0MS;	(* soft start waiting time*)
stControllerParameter.bStartUpRamping	:= FALSE;	(* enable ramping while start up phase *)
stControllerParameter.fWStartUpVeloPos	:= 1.0;	(* max gradient for increasing setpoint in start up phase*)
stControllerParameter.fWStartUpVeloNeg	:= 1.0;	(* max gradient for decreasing setpoint in start up phase *)

stControllerParameter.fWVeloPos	:= 1.0;	(* max gradient for increasing setpoint *)
stControllerParameter.fWVeloNeg	:= 1.0;	(* max gradient for decreasing setpoint *)

(* actual value parameters     												 *)
stControllerParameter.bFilter	:= FALSE;
stControllerParameter.tFilter	:= T#0S;

(* deadband parameters     													 *)
stControllerParameter.bDeadband	:= FALSE;
stControllerParameter.fEDeadband	:= 0.0; (* deadband *)

(* control value parameters     										 *)
stControllerParameter.fYMin		:= 0.0;	(* lower limit *)
stControllerParameter.fYMax		:= 100.0;	(* upper limit *)
stControllerParameter.fYManual	:= 0.0;	(* manual operation*)
stControllerParameter.fYOnFailure	:= 0.0;	(* control value on failure *)

stControllerParameter.tPWMCycleTime		:= T#1S;						(* PWM: period *)
stControllerParameter.tPWMMinOffTime	:= T#0S;						(* PWM: min off time *)
stControllerParameter.tPWMMinOnTime		:= T#0S;						(* PWM: min on time *)
stControllerParameter.tPWMWaitingTime	:= T#0S; (* PWM: min waiting time *)	(* not yet implemented !!!!*)

stControllerParameter.fYThresholdOff	:= 0.0;	(* 3-Point: Off threshold *)
stControllerParameter.fYThresholdOn		:= 0.0;	(* 3-Point: On threshold *)

stControllerParameter.nCyclesForSwitchOver := 100;

(* controller settings			     										 *)
stControllerParameter.bEnablePreController	:= FALSE;				(* enable precontroller *)
stControllerParameter.bEnableZones			:= FALSE;				(* enable zone around setpoint with open loop control *)
stControllerParameter.bEnableCVFilter		:= FALSE;				(* enable filter for CV (type see iFilterType) *)
stControllerParameter.iFilterType			:= E_FilterType_FIRSTORDER;	(* filtertype of CV filter *)
stControllerParameter.iControllerType		:= eCTRL_ControllerType_PID;	(* used controller normally PID *)

(* min max temperatures	     											 *)
stControllerParameter.TempLow			:= -50.0;
stControllerParameter.TempLowLow		:= -79.0;
stControllerParameter.TempHigh		:= 55.0;
stControllerParameter.TempHighHigh		:= 60.0;
stControllerParameter.TempAbsoluteHigh	:= 100.0;
stControllerParameter.TempAbsoluteLow	:= -77.0;

(* internal tuning parameters *)
stControllerParameter.fTuneKp	:= 1.2;
stControllerParameter.fTuneTn	:= 2.0;
stControllerParameter.fTuneTv	:= 0.42;
stControllerParameter.fTuneTd	:= 0.25;
```

**Comment by Barteling** [comment]
I did some digging,
You have some uncommon settings, which exposed some bugs, so thanks for that 👍🏼 

The timeout in this method:

```
(* general parameters 													 *)
stControllerParameter.iMode					:= 1;								(* 1=heating, 2=cooling, 3=heating&cooling *)
stControllerParameter.iReactionOnFailure	:= eCTRL_ReactionOnFailure_StopController;	(* 0=controller off, 1=manual op, 2=yMin, 3=yMax *)

stControllerParameter.bSelCtrlParameterSet 	:= FALSE; (* FALSE = internal set, TRUE = external set *)
..
..
..

```

Is caused by some bugs in STweep. I fixed them and they will be included in the next release, which will be end of august.
Short term fix: Change "Blank lines before multi line statement" from 2 to 1. Saves STweep a lot of thinking ^^

The invalid alignment in this PRG:

```
PROGRAM PRG_TestrigMonitoring
VAR

	fChamberPressure : REAL;

	tonFeedbackResponseTimer : TON;
	tFeedbackResponseTimeout : TIME := T#5S;
..
..
..

```

Is caused by your TabSize parameter, which is set 6, This should be equal to your editors Tab with value. Which is typically four. Making them equal will probably solve your issue.

Yes I do consider removing this parameter and just read the value which is used in the current environment. It's on my bucketlist ^^

![image](https://user-images.githubusercontent.com/33071638/182692744-67481ea5-77f9-44a5-b977-b35decc19339.png)
